n4js-libs/packages/n4js-mangelhaft-cli/src/n4js/org/eclipse/n4js/mangelhaft/runner/node/NodeTestAPI.n4js

```diff
@@ -199,7 +199,6 @@ export default public class NodeTestAPI {
         const args = NodeTestCLI.toCommandLine(options);
         args.unshift(consoleTestRunner.join("/"));
 
-        env.N4JS_RT_test_mode = "1";
```

Comment:
shouldn't `env.N4JS_RT_test_mode` be set somewhere?

Reply:
No, I want regular tests to fail when module loading fails. Just in case of stdlib spec tests (with not yet implemented API) this mode will be turned on.